src/clusterfuzz/_internal/metrics/logs.py

```diff
@@ -161,7 +161,7 @@ def format_record(record):
   }
 
   entry['location'] = getattr(record, 'location', {'error': True})
-  entry['extras'] = getattr(record, 'extras', {})
+  entry['extras'] = getattr(record, 'extras', None)
```

Comment:
This isn't correct.

{} on its own is a dict, not a set.

Reply:
you're right. seems like it's either a set or a bytes type that is being passed that is causing the error. `json.dumps()` seems to have `default` function that can handle this. updating this PR.